Dockerfile

```diff
@@ -1,23 +1,71 @@
-# AUTHOR: Roberto Vitillo
-# DESCRIPTION: Mozilla's Airflow container
-# BUILD: docker build --rm -t mozdata/telemetry-airflow
-# SOURCE: https://github.com/mozilla/telemetry-airflow
-
-FROM puckel/docker-airflow:1.7.1.3
-MAINTAINER vitillo
-
-USER root
-RUN apt-get update -yqq && \
-    apt-get install -yqq python-pip python-mysqldb && \
-    pip install boto3 && \
-    pip install airflow[async] && \
-    pip install airflow[password] && \
-    pip install retrying
-
-ADD ansible/files/airflow/airflow.cfg ${AIRFLOW_HOME}/airflow.cfg
-ADD ansible/files/airflow/entrypoint.sh ${AIRFLOW_HOME}/entrypoint.sh
-ADD ansible/files/airflow/replace_env.py ${AIRFLOW_HOME}/replace_env.py
-RUN chown airflow:airflow ${AIRFLOW_HOME}/airflow.cfg
-
-USER airflow
-ADD dags/ /usr/local/airflow/dags/
+FROM python:2-slim
+MAINTAINER Jannis Leidel <jezdez@mozilla.com>
+
+ENV PYTHONUNBUFFERED=1 \
+    AWS_REGION=us-west-2 \
+    # AWS_ACCESS_KEY_ID= \
+    # AWS_SECRET_ACCESS_KEY= \
+    SPARK_BUCKET=telemetry-spark-emr-2 \
+    AIRFLOW_BUCKET=telemetry-test-bucket \
+    PRIVATE_OUTPUT_BUCKET=telemetry-test-bucket \
+    PUBLIC_OUTPUT_BUCKET=telemetry-test-bucket \
+    EMR_KEY_NAME=mozilla_vitillo \
```

Comment:
Per discussion we're going to leave the README describing the production environment variables there, but take them out of the Dockerfile. The description of the env variables that need to be set will not necessarily be maintained, but there will be a link to the actual configs being used once the PR in cloudops-deployment is merged.